app/views/layouts/audit_wrapper.html.erb

```diff
@@ -10,6 +10,10 @@
   <%= audits_path %>
 <% end %>
 
+<% content_for :navbar_items do %>
```

Comment:
Do we need this here?

Reply:
Yes. It overrides the nav bar items in the base layout so they don't appear in the audit tool. It's the same approach we have taken to overriding the app name etc.